src/main/resources/application.yml

```diff
@@ -0,0 +1,9 @@
+spring:
+  application:
+    name: loadflow-server
+
+server:
+  port: 5008
+
+loadflow-server:
```

Comment:
Would it be network-store-server config instead?

Reply:
Corrected